### ISSUE TITLE
Error with newest plugin ver

### ISSUE THREAD

**Issue by technotherion** [Menithal/Blender-Hifi-Addon#7]
```
Traceback (most recent call last):
  File "C:\Users\OrangeTwo\AppData\Roaming\Blender Foundation\Blender\2.78\scripts\addons\Blender-Hifi-Addon-master\hifi_json_writer.py", line 409, in execute
    bpy.ops.object.select_all(action = 'DESELECT')
  File "C:\Program Files\Blender Foundation\Blender\2.78\scripts\modules\bpy\ops.py", line 189, in __call__
    ret = op_call(self.idname_py(), None, kw)
RuntimeError: Operator bpy.ops.object.select_all.poll() failed, context is incorrect

location: <unknown location>:-1
```

Attempting to export a simple blender shape

**Comment by Menithal** [comment]
This one was with the 0.4.4 version?

**Comment by Menithal** [comment]
Ok Found it, Caused by being in any other mode than Object mode, putting a safeguard in

**Comment by Menithal** [comment]
Fixed in 0.4.5
